.rubocop.yml

```diff
@@ -22,7 +22,12 @@ Metrics/BlockLength:
 Metrics/LineLength:
   Exclude:
     - 'Guardfile'
-  Max: 110
```

Comment:
I vote for `Enabled: false`.

Reply:
Why disable line length? The intent was to enforce a line length that is a bit more accessible for folks that need it

Reply:
Yes that's right. We discussed this in another PR. Ignore me.